ablySpec/RestClient.swift

```diff
@@ -326,6 +327,82 @@ class RestClient: QuickSpec {
                 }
             }
 
+            // RSC14c
```

Comment:
Odd, I made this yesterday [RestClient 3cb5f8c](https://github.com/ably/ably-ios/commit/cfd4b3725a150a3a86744778d819c45115a3d2e8#diff-4ed1dc4a695e1026db6d7a9f33cb5f8c)! And I updated my branch with the latest merges.